content/docs/start/metrics-parameters-plots.md

```diff
@@ -215,24 +218,24 @@ We are definitely not happy with the AUC value we got so far! Let's edit the
 +  ngrams: 2
 ```
 
-And the beauty of `dvc.yaml` is that all you need to do now is to run:
+The beauty of `dvc.yaml` is that all you need to do now is run:
 
 ```dvc
 $ dvc repro
 ```
 
-It'll analyze the changes, use existing cache of previous runs, and execute only
-the commands that are needed to get the new results (model, metrics, plots).
+It'll analyze the changes, use existing <abbr>run-caches</abbr>, and execute
+only the commands which are needed to produce new results (model, metrics,
+plots).
 
 The same logic applies to other possible adjustments — edit source code, update
-datasets — you do the changes, use `dvc repro`, and DVC runs what needs to be
-run.
+datasets — you do the changes, use `dvc repro`, and DVC runs what needs to be.
 
 ## Comparing iterations
 
 Finally, let's see how the updates improved performance. DVC has a few commands
-to see metrics and parameter changes, and to visualize plots, for one or more
-pipeline iterations. Let's compare the current "bigrams" run with the last
+to see metrics & parameter changes and to visualize plots (for one or more
+pipeline iterations). Let's compare the current "bigrams" run with the last
```

Comment:
The punctuation change here is a bit misleading. There's a command to viz metrics, another one for params, and another one for plots. Now it looks as though maybe there's just one for metrics and params (not sure about using `&` either), and one for plots.

And yes `exp diff` includes metrics and params but we're not referring to that here 🙂 (they will all get merged probably anyway)

Reply:
```suggestion
to see changes in and visualize metrics, parameters, and plots (for one or more
pipeline iterations). Let's compare the current "bigrams" run with the last
```

Reply:
"see changes in and visualize" - too complex language IMO, we want to keep sentences simple 🙂